src/edu/csus/ecs/pc2/core/imports/clics/CLICSContests.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+
+/**
+ * A JSON representation for the CLICS API contests json data.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+public class CLICSContests {
+
+    // type contests data = {duration=5:00:00.000, start_time=2022-05-21T18:00:00.000+00,
+    // scoreboard_freeze_duration=1:00:00.000, name=nac22practice5, id=nac22practice5, formal_name=null, penalty_time=20}
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private String duration;
+
+    @JsonProperty
+    private String start_time;
+
+    @JsonProperty
+    private String scoreboard_freeze_duration;
+
+    @JsonProperty
+    private String formal_name;
+
+    @JsonProperty
+    private String penalty_time;
+
```

Comment:
The above collection of attributes does not cover everything listed in the CLICS 2202-07 specification.  For example, the spec also defines that a `contest` object may contain `countdown_pause_time`, `scoreboard_type`, `banner`, `logo`, `location.latitude`, and `location.longitude`.  Since we are trying to bring PC2 code up to CLICS specs, I think this class should at least declare all of the objects (fields) defined in the spec.

Reply:
Out of scope, added as #559

Reply:
I disagree that this is "out of scope".  The tool is supposed to read a CLICS 2202-07 event feed.  This event feed could legitimately contain any or all of the fields which are currently missing in the class definition.  It seems to me that including every field that could be in the event feed. is an important component of implementing this class (and all the other similar classes about which I've made equivalent comments.

Said another way:  the event feed COULD contain those fields; what happens in the implementation if it does?

Reply:
Since those fields are not defined in the class, they are ignored.
Those fields can be added later when a review of the actual comparison output is done.
Rather than include those fields in the comparison now, the layout/content of the comparison
should be done first.    After that the fields described in #559 can be added once
the format/content has been agreed upon.